src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: $gap;
+  margin: 0 auto;
```

Comment:
why did you remove max-width? your cards are now not centered
![image](https://user-images.githubusercontent.com/37161315/170268752-3989ef65-6cd2-4e68-8a10-ae8d8d6e077f.png)